<launch>
  <node pkg="ford_project" type="exp_control" name="exp_control">
  	<param name="data_file_path" value="$(env HOME)/Desktop/data"/>
  	<param name="protocal_file_path" value="$(find ford_project)/protocals"/>
  </node>
  <!--node pkg="ford_project" type="gesture_DTW" name="gesture_detector">
  	<param name="gesture_data_path" value="$(find ford_project)/gesture_data"/>
  </node-->
</launch>
